.circleci/config.yml

```diff
@@ -18,13 +18,13 @@ parameters:
   # Configurable Versions
   cli-version:
     type: string
-    default: '7.45.1'
+    default: '7.132.0'
```

Comment:
Great job for finding and updating this. We need to discuss (with team) if we `latest` here so that the `npm install -g sfdx-cli@<< pipeline.parameters.cli-version >>` command below can use the latest released version of cli.

Reply:
@RitamAgrawal mentions that 7.45.1 is probably the version that uses toolbelt .. maybe that has something to do with this version. But why is the cli installed only in windows and how is it used?

Reply:
If 7.45.1 uses toolbelt, this is a somewhat significant change and would deserve a separate PR. We run the integration tests in windows and probably that's why we set this only in windows.

Reply:
Noted during 11sies, I'll revert this change out of this PR, and make a separate WI/PR for it.

Reply:
Also smart callout about the toolbelt crossover @RitamAgrawal 🎉